libs/pyTermTk/TermTk/TTkCore/ttk.py

```diff
@@ -106,11 +106,15 @@ def __init__(self, *,
         self._mouseCursor = None
         self._showMouseCursor = os.environ.get("TERMTK_MOUSE",mouseCursor)
         super().__init__(**kwargs)
+
+        # Register events
         TTkInput.inputEvent.connect(self._processInput)
         TTkInput.pasteEvent.connect(self._processPaste)
         TTkSignalDriver.sigStop.connect(self._SIGSTOP)
         TTkSignalDriver.sigCont.connect(self._SIGCONT)
         TTkSignalDriver.sigInt.connect( self._SIGINT)
+        TTkSignalDriver.init()
```

Comment:
I have the init in the main loop because I don't want to capture the events when you create the TTk root object but only in the runtime.